packages/Card/package.json

```diff
@@ -20,6 +20,9 @@
     "@paprika/helpers": "^2.2.0-next.1",
     "@paprika/stylers": "^1.1.4-next.0",
     "@paprika/tokens": "^1.1.3-next.0",
+    "@paprika/avatar": "^2.2.11-next.2",
+    "@paprika/button": "^1.1.9-next.3",
+    "@paprika/tag": "^0.0.18-next.4",
```

Comment:
None of the Card src files actually imports any of these components – we just show how they are used together in the Storybook stories.  So they don't need to be dependencies.